File: BackendAPI/Application/DTOs/Output/OrderProductOutputDto.cs
namespace Application.DTOs.Output;

public class OrderProductOutputDto
{
    public required Guid ProductId { get; set; }
    public required string ProductName { get; set; }
    public required string ProductDescription { get; set; }
    public required string ProductImageUrl { get; set; }
    public required string CompanyName { get; set; }
    public required decimal? MinimalPrice { get; set; }
    public required int Quantity { get; set; }
    public required decimal PriceAtPurchase { get; set; }
    public required DateTimeOffset OrderedAt { get; set; }
}

File: FrontendWeb/src/declarations/dtos/output/OrderProductOutputDto.ts
// OrderProductOutputDto.ts
export interface OrderProductOutputDto {
	productId: string;
	productName: string;
	productDescription: string;
	productImageUrl: string;
	companyName: string;
	quantity: number;
	priceAtPurchase: number;
	minimalPrice?: number;
	orderedAt: string;
}
